### PR TITLE
Ensure children isn't null before accepting

### PR DESCRIPTION
**Pull request by mit-mit**
Fixes https://github.com/dart-lang/markdown/issues/181

### CODE REVIEW

**Comment by mit-mit** [comment]
cc @kasperl

File: pubspec.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: markdown
-version: 1.0.0-dev
+version: 1.0.1
```
Comment (kevmoo) [inline pubspec.yaml]: haven't released 1.0.0 yet. So leave it at dev

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## 1.0.1
```
Comment (kevmoo) [inline CHANGELOG.md]: put this under 1.0.0 – hasn't been released

**Comment by kevmoo** [comment]
Also, see why travis is broken...

File: pubspec.yaml (hunk 1 above)
Comment (mit-mit) [reply pubspec.yaml]: why have we not released 1.0.0 yet?

**Comment by mit-mit** [comment]
Travis should be resolved in https://github.com/dart-lang/markdown/pull/182/commits/bee2c73b255ae190f6ebac96d824eb11d33686ad

File: pubspec.yaml (hunk 1 above)
Comment (kevmoo) [reply pubspec.yaml]: Because...it hasn't...haven't got around to it.

Comment (mit-mit) [reply pubspec.yaml]: Isn't it just take this PR and hit publish?

**Comment by kevmoo** [comment]
Sure!

From mobile. Sorry for brevity and/or spelling.

**Comment by mit-mit** [comment]
Travis is green now

**Comment by kevmoo** [comment]
Fix the bits so this is still 1.0.0-dev...I'll do a release roll...

**Comment by mit-mit** [comment]
Still not sure I full understand how you guys manage these packages, but anywho I reverted the version edits

**Comment by mit-mit** [comment]
But please make sure the fix is published on something I can do a version constraint for from the `flutter_markdown` plugin

**Pull request merged**